presto-docs/src/main/sphinx/functions.rst

```diff
@@ -29,3 +29,4 @@ Functions and Operators
     functions/color
     functions/session
     functions/teradata
+    functions/ipaddress
```

Comment:
Should we move this to be above `functions/color`?

Reply:
@rongrong Why do we need to move it above `functions/color`?

Reply:
@rongrong I don't know the exact reason but made the change. Now the html looks like this

![image](https://user-images.githubusercontent.com/33299678/66788422-5c28d700-ee9c-11e9-815d-b0fd61822274.png)

Reply:
It should be before `session` and `teradata` because that's very different category. I don't have strong opinions on before or after color. Now thinking about this, should we just call these "IP Functions"?